### ISSUE TITLE
Could you please provide complete data?

### ISSUE THREAD

**Issue by ysgncss** [xmshi-trio/MSL#1]

**Comment by xmshi-trio** [comment]
Hi, thanks for your attention for our work. The work was finished when I was in my internship in Tencent, and according to the requirement of Tencent, you need contact Haifeng Hu (simbahu@tencent.com) to get the complete data.

**Comment by squareRoot3** [comment]
However, Haifeng Hu did not give any response after I asked him the data by email.

**Comment by xmshi-trio** [comment]
Hi, I have send a message to him, and he will send you the data later.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"sQrt3"<notifications@github.com&gt;;
发送时间:&nbsp;2020年3月11日(星期三) 下午3:33
收件人:&nbsp;"xmshi-trio/MSL"<MSL@noreply.github.com&gt;;
抄送:&nbsp;"Trickster"<1669490794@qq.com&gt;;"Comment"<comment@noreply.github.com&gt;;
主题:&nbsp;Re: [xmshi-trio/MSL] Could you please provide complete data? (#1)




However, Haifeng Hu did not give any response after I asked him the data by email.

**Comment by squareRoot3** [comment]
@xmshi-trio sorry to bother you again but I have not received the data yet. My email address is sqrt3tjh@gmail.com . I have sent emails to both you and Hu many months ago. 
It's understandable that the data release is difficult due to the property issue of the company, so I only want a clear response or a clear rejection.

**Comment by xmshi-trio** [comment]
Hi, I am really sorry that Haifeng Hu is always very busy with his work, and I have reminded him again. I think he should send you the data now. Please check your email.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"sQrt3"<notifications@github.com&gt;;
发送时间:&nbsp;2020年3月26日(星期四) 晚上11:43
收件人:&nbsp;"xmshi-trio/MSL"<MSL@noreply.github.com&gt;;
抄送:&nbsp;"Trickster"<1669490794@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [xmshi-trio/MSL] Could you please provide complete data? (#1)





 
@xmshi-trio sorry to bother you again but I have not received the data yet. My email address is sqrt3tjh@gmail.com . I have sent emails to both you and Hu many months ago.
 It's understandable that the data release is difficult due to the property issue of the company, so I only want a clear response or a clear rejection.

**Comment by squareRoot3** [comment]
I have received the data. Thank you very much!  
**For other readers: please do not ask me for the data.**

**Comment by xmshi-trio** [comment]
You are welcome. If you have any question about paper or data, please do not hesitate to contact us.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"sQrt3"<notifications@github.com&gt;;
发送时间:&nbsp;2020年3月27日(星期五) 下午3:00
收件人:&nbsp;"xmshi-trio/MSL"<MSL@noreply.github.com&gt;;
抄送:&nbsp;"Trickster"<1669490794@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [xmshi-trio/MSL] Could you please provide complete data? (#1)





 
I have received the data. Thank you very much!

**Comment by ray075hl** [comment]
@xmshi-trio  Very appreciate you share this great idea and code. Could you send me a complete data which this project need?  Thank you very much!  my email address is **729872400@qq.com**. 

Futhermore, I have some questions about your paper.
**1.**  How can I get target y_{i}  when model at **weak supervision for pre-training stage**?
**2.**  x_{i} is a embeding of a word or a sentence ?
![image](https://user-images.githubusercontent.com/23446542/78766505-ec635700-79bb-11ea-9a25-339e56066505.png)
***
![image](https://user-images.githubusercontent.com/23446542/78767458-36007180-79bd-11ea-9942-289f7b1f7b47.png)

**Comment by xmshi-trio** [comment]
> @xmshi-trio Very appreciate you share this great idea and code. Could you send me a complete data which this project need? Thank you very much! my email address is **[729872400@qq.com](mailto:729872400@qq.com)**.
> 
> Futhermore, I have some questions about your paper.
> **1.** How can I get target y_{i} when model at **weak supervision for pre-training stage**?
> **2.** x_{i} is a embeding of a word or a sentence ?
> ![image](https://user-images.githubusercontent.com/23446542/78766505-ec635700-79bb-11ea-9a25-339e56066505.png)
> 
> ![image](https://user-images.githubusercontent.com/23446542/78767458-36007180-79bd-11ea-9942-289f7b1f7b47.png)

Hi, thanks for your attention for our work. The work was finished when I was in my internship in Tencent, and according to the requirement of Tencent, you need contact Haifeng Hu (simbahu@tencent.com) to get the complete data. He is always busy with his work, and forgets to reply the email. Please be patient.

Question 1: weak supervision comes from doctors response when pertaining.
Question 2: x_i is the i th word. I make the error when writing the paper. Thanks for your question.

If you have any question about the paper or code, I am grad to hear from you.

**Comment by ray075hl** [comment]
@xmshi-trio   Thank you for your  reply！
**So, the pretraining label generated by exact match by each standard medical concept?**

**Comment by xmshi-trio** [comment]
> @xmshi-trio Thank you for your reply！
> **So, the pretraining label generated by exact match by each standard medical concept?**

Yes!

**Comment by 460130107** [comment]
@xmshi-trio Very appreciate you share your work.
Could you send me the complete data? 
my email address is 460130107@qq.com
Thank you very much! ！！
十分感谢

**Comment by xmshi-trio** [comment]
> @xmshi-trio Very appreciate you share your work.
> Could you send me the complete data?
> my email address is [460130107@qq.com](mailto:460130107@qq.com)
> Thank you very much! ！！
> 十分感谢

Hi, thanks for your attention for our work. The work was finished when I was in my internship in Tencent, and according to the requirement of Tencent, you need contact Haifeng Hu (simbahu@tencent.com) to get the complete data. He is always busy with his work, and forgets to reply the email. Please be patient.

**Comment by wjczf123** [comment]
My email address is 1044795134@qq.com . I have sent emails to Hu 20 days ago. Can you send him an email and ask him to send me dataset?

**Comment by xmshi-trio** [comment]
Hi,
I am sorry for the late response. I have asked Haifeng Hu to send you the dataset as soon as possible.&nbsp;
If you have any question about the paper or the dataset, I am happy to hear from you.


Xiaoming Shi,
Harbin Institution of Technology,
Tencent.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "xmshi-trio/MSL"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年10月10日(星期六) 晚上11:36
收件人:&nbsp;"xmshi-trio/MSL"<MSL@noreply.github.com&gt;;
抄送:&nbsp;"Trickster"<1669490794@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [xmshi-trio/MSL] Could you please provide complete data? (#1)





 
My email address is 1044795134@qq.com . I have sent emails to Hu 20 days ago. Can you send him an email and ask him to send me dataset?

**Comment by ysgncss** [comment]
God, I've almost forgotten which paper this is！Could you please send me the data？ 2207589533@qq.com

**Comment by xmshi-trio** [comment]
Hi,&nbsp;


I am really sorry for my late response.&nbsp;
The data and the code are available. The project homepage is https://github.com/xmshi-trio/MSL.git.
According to Tencent's regulations, the data can only be used for research purposes.
If you have any question about the data or the paper, please do not&nbsp;hesitate to contact us. We are delighted to hear from you.


Xiaoming Shi,
Research Center for Social Computing and Information Retrieval,
Harbin Institute of Technology.


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "xmshi-trio/MSL"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年10月13日(星期二) 下午4:45
收件人:&nbsp;"xmshi-trio/MSL"<MSL@noreply.github.com&gt;;
抄送:&nbsp;"Trickster"<1669490794@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [xmshi-trio/MSL] Could you please provide complete data? (#1)





 
God, I've almost forgotten which paper this is！Could you please send me the data？ 2207589533@qq.com

**Comment by xmshi-trio** [comment]
Hi,&nbsp;


I am really sorry for my late response.&nbsp;
The data and the code are available. The project homepage is https://github.com/xmshi-trio/MSL.git.
According to Tencent's regulations, the data can only be used for research purposes.
If you have any question about the data or the paper, please do not&nbsp;hesitate to contact us. We are delighted to hear from you.


Xiaoming Shi,
Research Center for Social Computing and Information Retrieval,
Harbin Institute of Technology.








------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "xmshi-trio/MSL"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年4月20日(星期一) 下午5:27
收件人:&nbsp;"xmshi-trio/MSL"<MSL@noreply.github.com&gt;;
抄送:&nbsp;"Trickster"<1669490794@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [xmshi-trio/MSL] Could you please provide complete data? (#1)





 
@xmshi-trio Very appreciate you share your work.
 Could you send me the complete data?
 my email address is 460130107@qq.com
 Thank you very much! ！！
 十分感谢

**Comment by xmshi-trio** [comment]
Hi,&nbsp;


I am really sorry for my late response.&nbsp;
The data and the code are available. The project homepage is https://github.com/xmshi-trio/MSL.git.
According to Tencent's regulations, the data can only be used for research purposes.
If you have any question about the data or the paper, please do not&nbsp;hesitate to contact us. We are delighted to hear from you.


Xiaoming Shi,
Research Center for Social Computing and Information Retrieval,
Harbin Institute of Technology.








------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "xmshi-trio/MSL"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年10月10日(星期六) 晚上11:36
收件人:&nbsp;"xmshi-trio/MSL"<MSL@noreply.github.com&gt;;
抄送:&nbsp;"Trickster"<1669490794@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [xmshi-trio/MSL] Could you please provide complete data? (#1)





 
My email address is 1044795134@qq.com . I have sent emails to Hu 20 days ago. Can you send him an email and ask him to send me dataset?

**Comment by xmshi-trio** [comment]
Hi,


I am really sorry for my late response.
The data and the code are available. The project homepage is&nbsp;https://github.com/xmshi-trio/MSL.git.
According to Tencent's regulations, the data can only be used for research purposes.
If you have any question about the data or the paper, please do not hesitate to contact us. We are delighted to hear from you.


Xiaoming Shi,
Research Center for Social Computing and Information Retrieval,
Harbin Institute of Technology.






------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "xmshi-trio/MSL"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年4月8日(星期三) 下午5:01
收件人:&nbsp;"xmshi-trio/MSL"<MSL@noreply.github.com&gt;;
抄送:&nbsp;"Trickster"<1669490794@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [xmshi-trio/MSL] Could you please provide complete data? (#1)





 
@xmshi-trio  Very appreciate you share this great idea and code. Could you send me a complete data which this project need?  Thank you very much!  my email address is 729872400@qq.com

**Comment by xmshi-trio** [comment]
Hi,


I am really sorry for my late response.
The data and the code are available. The project homepage is&nbsp;https://github.com/xmshi-trio/MSL.git.
According to Tencent's regulations, the data can only be used for research purposes.
If you have any question about the data or the paper, please do not hesitate to contact us. We are delighted to hear from you.


Xiaoming Shi,
Research Center for Social Computing and Information Retrieval,
Harbin Institute of Technology.






------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "xmshi-trio/MSL"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年3月27日(星期五) 下午3:00
收件人:&nbsp;"xmshi-trio/MSL"<MSL@noreply.github.com&gt;;
抄送:&nbsp;"Trickster"<1669490794@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [xmshi-trio/MSL] Could you please provide complete data? (#1)





 
I have received the data. Thank you very much!

**Comment by xmshi-trio** [comment]
Hi,


I am really sorry for my late response.
The data and the code are available. The project homepage is&nbsp;https://github.com/xmshi-trio/MSL.git.
According to Tencent's regulations, the data can only be used for research purposes.
If you have any question about the data or the paper, please do not hesitate to contact us. We are delighted to hear from you.


Xiaoming Shi,
Research Center for Social Computing and Information Retrieval,
Harbin Institute of Technology.









------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "xmshi-trio/MSL"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年3月7日(星期六) 中午12:13
收件人:&nbsp;"xmshi-trio/MSL"<MSL@noreply.github.com&gt;;
抄送:&nbsp;"Subscribed"<subscribed@noreply.github.com&gt;;
主题:&nbsp;[xmshi-trio/MSL] Could you please provide complete data? (#1)
